### PR TITLE
SITES-000: Do not sync aggregated CSS/JS

### PR DESCRIPTION
**Pull request by jbickar**
# READY FOR REVIEW

# Summary
- Exclude aggregated CSS and JS

# Needed By (Date)
- Whenever

# Criticality
- How critical is this PR on a 1-10 scale? 2/10
- It came about as a request from @cjwest in troubleshooting some of the PICAL sites

# Steps to Test

1. Check out this branch
2. Migrate a site
3. Look in the CSS and JS directories and verify that they are empty
4. Visit the homepage
5. Verify that aggregated CSS and JS are generated

# Affected Projects or Products
- ACSF

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by jbickar** [comment]
@sherakama, any thoughts on this? Testing now.

**Comment by cjwest** [comment]
👏

**Comment by sherakama** [comment]
This seems reasonable as long as we clear out all the caches, which we do, often.

File: roles/post-db-restore/tasks/main.yml [hunk 1]
```diff
@@ -136,7 +136,7 @@
 
 # Copy public files from AFS so that they exist and Drupal can find them if needed
 - name: Copy public files from AFS to Site Factory
-  shell: "drush -y rsync '/afs/ir/dist/drupal/{{ site_prefix }}_{{ inventory_hostname }}/files/' @acsf{{ drush_environment }}.{{ stack }}.{{ acsf_site_name }}:%files/ --exclude-paths=sites/default/files/js_injector/"
+  shell: "drush -y rsync '/afs/ir/dist/drupal/{{ site_prefix }}_{{ inventory_hostname }}/files/' @acsf{{ drush_environment }}.{{ stack }}.{{ acsf_site_name }}:%files/ --exclude-paths=sites/default/files/js_injector/ --exclude-paths=sites/default/files/css/ --exclude-paths=sites/default/files/js/"
```
Comment (sherakama) [inline roles/post-db-restore/tasks/main.yml]: Shouldn't the `--exclude-paths` come before the rsync option?
https://drupal.stackexchange.com/questions/178224/clear-example-for-drush-rsync-exclude-paths

Comment (jbickar) [reply roles/post-db-restore/tasks/main.yml]: Like I'm going to believe what Greg Anderson has to say about `drush rsync`

Comment (sherakama) [reply roles/post-db-restore/tasks/main.yml]: Also, shouldn't the options be separated by `:`

Comment (jbickar) [reply roles/post-db-restore/tasks/main.yml]: How do you mean?

Comment (sherakama) [reply roles/post-db-restore/tasks/main.yml]:
```suggestion
  shell: "drush -y  --exclude-paths=sites/default/files/js_injector/ --exclude-paths=sites/default/files/css/:sites/default/files/js/ rsync '/afs/ir/dist/drupal/{{ site_prefix }}_{{ inventory_hostname }}/files/' @acsf{{ drush_environment }}.{{ stack }}.{{ acsf_site_name }}:%files/"
```

Comment (jbickar) [reply roles/post-db-restore/tasks/main.yml]: I wasn't aware of that shorthand. Why not all three?

Comment (sherakama) [reply roles/post-db-restore/tasks/main.yml]: Gotta leave something for you.

Comment (jbickar) [reply roles/post-db-restore/tasks/main.yml]: Can't argue with that logic.

**Pull request merged**